component/data-view/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
data-view component from native-x is not refreshing child view if data changes. That's why a fixed version is added to local repo.